node_modules/oae-mixpanel/tests/test-users.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var assert = require('assert');
+
+var RestAPI = require('oae-rest');
+var TestsUtil = require('oae-tests');
+
+var MixPanelTestUtil = require('oae-mixpanel/lib/test/util');
+
+describe('MixPanel - Users', function() {
```

Comment:
Rather than scoping with text conventions like a dash, we can instead use nested `describe` blocks:

```javascript
describe('MixPanel', function() {
    describe('Users', function() {
```